binder-index.md

```diff
@@ -188,34 +164,59 @@ These are noted in the README.md files for each sample, along with complete inst
       <td></td>
     </tr>
     <tr>
-      <td><strong>Azure Quantum service:</strong></td>
-      <td><strong><a href="./samples/azure-quantum/chemistry/README.md">Chemistry</a></strong></td>
+      <th>Azure Quantum service:</th>
+      <td rowspan="1"><strong><a href="./samples/getting-started/azure-quantum/README.md">Getting started</a></strong></td>
```

Comment:
I think it's a better fit here, since this makes it actionable for users looking samples about the service to quickly see how to get started. Mixing in service-oriented samples with all other quantum computing samples could make them harder to find.

Reply:
They are "Getting started" samples though, if I were a user I would expect to find everything related to how to get started under "Getting Started".